images/runtime/python/3.10/bullseye.Dockerfile

```diff
@@ -81,6 +81,7 @@ LABEL io.buildpacks.stack.id="oryx.stacks.skeleton"
 RUN ${IMAGES_DIR}/runtime/python/install-dependencies.sh
 RUN pip install --upgrade pip \
     && pip install gunicorn \
+    && pip install uvicorn \
```

Comment:
Unfortunately, I don't think we can add  these for the existing images as it can have some implications in terms of size and side-effects for existing customers.
The easiest ways I would see is:
* Waiting for a new Python version and only enable this here.
* Create a new image just for this case, which is a bit annoying as well.
We can sync offline if you want.